handler/handler.go

```diff
@@ -67,6 +71,25 @@ func (h *ContentPublishedHandler) Handle(ctx context.Context, event *models.Cont
 	}
 
 	log.Info(ctx, "event successfully handled", logData)
-
 	return nil
 }
+
+// incoming keywords validation
+func ValidateKeywords(keywords []string) []string {
+
+	var strArray []string
+	validKeywords := make([]string, 0)
+
+	for i := range keywords {
+		strArray = strings.Split(keywords[i], ",")
+		for j := range strArray {
+			validKeywords = append(validKeywords, strArray[j])
+		}
+	}
+
+	if len(validKeywords) < 5 {
+		return validKeywords
+	} else {
+		return validKeywords[:5]
+	}
```

Comment:
Can you update this so max valid keyring size is var or const please.

Also `else` not needed here you can just do:
```
    if len(validKeywords) < 5 {
        return validKeywords
    }
    return validKeywords[:5]
```